Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -355,7 +357,8 @@ void JitArm64::fcmpx(UGeckoInstruction inst)
 
 	FixupBranch pNaN1, pNaN2, pNaN3, pLesser, pGreater;
 	FixupBranch continue1, continue2, continue3;
-	ORR(XA, ZR, 32, 0, true);
+
+	m_float_emit.ORR(XA, ZR, 32, 0, true);
```

Comment:
Yeah, well, this pr is way off, but I am studying the fcmp(o/u) functions of the ARMv8-A1 and the Broadway microprocessor to rewrite the jitarm64 fcmpx function as it currently doesn't seem to be doing it for all games.